### PR TITLE
Added default legends to line and scatter graphs [re #5457]

### PR DESCRIPTION
**Pull request by lshrinivas**
As the name says - called out things on interest inline below

### CODE REVIEW

File: test/presenter/Chart.spec.js [hunk 1]
```diff
@@ -651,7 +651,7 @@ function (
...
...;"><line x2="7" y2="0"></line><text dy=".32em" x="-7" y="0" style="text-anchor: end;">90</text></g><g class="tick" transform="translate(0,72.0754716981132)" style="opacity: 1;"><line x2="7" y2="0"></line><text dy=".32em" x="-7" y="0" style="text-anchor: end;">100</text></g><g class="tick" transform="translate(0,32.15094339622641)" style="opacity: 1;"><line x2="7" y2="0"></line><text dy=".32em" x="-7" y="0" style="text-anchor: end;">110</text></g><path class="domain" d="M0,20H0V480H0"></path></g><g class="zoom_fixture_x"></g><g class="zoom_fixture_y"></g><svg class="legend" x="110" y="30" width="760" height="30"><rect class="legend_bg" width="100%" height="100%"></rect><g class="legend_group"><rect x="10" y="14" width="12" height="2" fill="green" class="legend_swatch"></rect><text x="32" y="20" font-size="14" class="legend_label">graph_3</text></g><g class="legend_group"><rect x="87.875" y="14" width="12" height="2" fill="#09e" class="legend_swatch"></rect><text x="109.875" y="20" font-size="14" class="legend_label">graph_4</text></g></svg><line x1="100" x2="880" y1="20" y2="20" class="bounding_box"></line><line x1="880" x2="880" y1="20" y2="480" class="bounding_box"></line></svg>';
```
Comment (lshrinivas) [inline test/presenter/Chart.spec.js:655]: This tests that the default legend got added

**Comment by lshrinivas** [comment]
All tests pass

**Comment by nickdoesdesign** [comment]
Looks good to me. Maybe a small test for this?

**Comment by lshrinivas** [comment]
There is - the diff in Chart smoke test has the new legend showing up

**Comment by choochburger** [comment]
Yeah, [LegendView.spec.js](https://github.com/nutonian/nugget/blob/p_legends/test/views/LegendView.spec.js) has good general coverage, too. Ship it! :+1:

**Pull request merged**